modules/system/defaults/NSGlobalDomain.nix

```
@@ -216,6 +216,14 @@ in {
       '';
     };
 
+    system.defaults.NSGlobalDomain.NSWindowShouldDragOnGesture = mkOption {
+      type = types.nullOr types.bool;
+      default = null;
+      description = ''
+      	Whether to enable moving window by holding anywhere on it like on Linux. The default is null.
```

```suggestion
      	Whether to enable moving window by holding anywhere on it like on Linux. The default is false.
```

When listing the default value for `defaults` options in `nix-darwin` the convention is to specify the default value of the key in macOS rather than the default value of the `nix-darwin` option (since it’s always `null`).

---

> When listing the default value for `defaults` options in `nix-darwin` the convention is to specify the default value of the key in macOS rather than the default value of the `nix-darwin` option (since it’s always `null`).


I noticed this message now, sorry. And I didn't know that I should write `false` instaed of `null`. Thank you.